Libraries/LibCore/ProcessStatisticsReader.cpp

```diff
@@ -108,6 +109,43 @@ HashMap<pid_t, Core::ProcessStatistics> ProcessStatisticsReader::get_all()
     return map;
 }
 
+Optional<String> ProcessStatisticsReader::get_current_working_directory(pid_t pid)
```

Comment:
`String` already has a null state and we use that instead of wrapping them in `Optional`.